packages/@expo/metro-config/src/transform-worker/browserslist.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright © 2025 650 Industries.
+ */
+
+const debug = require('debug')('expo:metro:browserslist') as typeof console.log;
+
+const browserslistCache: Record<string, import('lightningcss').Targets> = {};
+
+export async function getBrowserslistTargets(
+  projectRoot: string
+): Promise<import('lightningcss').Targets> {
+  if (browserslistCache[projectRoot]) {
+    return browserslistCache[projectRoot];
+  }
+  const browserslist = await import('browserslist');
+  const { browserslistToTargets } = require('lightningcss') as typeof import('lightningcss');
```

Comment:
nit: mixing `require` and `await import` here seems odd. `browserslist` is CommonJS-only so this might also not be necessary